HadesScheduler/docker/docker.go

```diff
@@ -27,6 +27,8 @@ type DockerConfig struct {
 	DockerHost           string `env:"DOCKER_HOST" envDefault:"unix:///var/run/docker.sock"`
 	ContainerAutoremove  bool   `env:"CONTAINER_AUTOREMOVE" envDefault:"true"`
 	DockerScriptExecutor string `env:"DOCKER_SCRIPT_EXECUTOR" envDefault:"/bin/bash -c"`
+	CPU_limit            uint   `env:"DOCKER_CPU_LIMIT"`
+	RAM_limit            string `env:"DOCKER_RAM_LIMIT"`
```

Comment:
I guess it is a good idea to add a comment with the intended format.
```suggestion
	CPU_limit            uint   `env:"DOCKER_CPU_LIMIT"` // Number of CPUs - e.g. '6'
	RAM_limit            string `env:"DOCKER_RAM_LIMIT"` // RAM usage in GB or MB  - e.g. '4GB' 
```

Reply:
I've changed the logic and also the naming. Not it supports the memory limit similarly to Docker itself (e.g. `4g` and `4m`)